### DISCUSSION TITLE
June 1st, 2022 (Prashant, Gonzalo, Tom Mitchel, and Pedro)

### DISCUSSION THREAD

**Discussion by Fontanapink** [SynBioDex/SBOL-Editors#5] [Editors meetings]
**Agenda:**
**Website (James S-B)**
- Some website issues still require some attention

**SEPs / Github issues (Prashant)**
- SEP 54 discussion came to a point where it is not going to be a part of SBOL, Jake is going to make the changes.
- SEP 55 is moving to examples
- Prashant is going to "close" the issue so that they won't appear any longer on the issue list. 

**Communications (Gonzalo)**
- Gonzalo sent the announcement last week. 
- Some point we have to make the videos on our work on HARMONY 2022 work to show what happened
- Prashant is proposing if it would be productive/prudent to have BDAC producing videos that use SBOL and put them on youtube.
- Gonzalo: What if we make the videos on the SBOL youtube channel, and have them listed in BDAC

**Report on other SBOL meetings (Tom)**
- No other SBOL meetings
- Next week Steering committee and SBOL3

**Any Other Business**
- Talked about moving the websites in http://co.mbine.org/admin/build/path before the server goes down. 
- Talked a little about IWBDA 2022
- Prashant has an incomplete README file he is going to push to Github Examples repositories for review

**Comment by Gonza10V** [comment]
I think we need to make a template for short videos (10-15 minutes) with intro, content and outro. Similar to the workshop prepared at HARMONY and presented at SEED 2022, each video should work as stand alone tutorial and the playlist provide the DBTL iteration.

**Reply by cjmyers** [reply]
Agreed
